src/v2/services/companies.js

```diff
@@ -102,17 +102,25 @@ const addAddress = async (companyId, addressId, roleName, data = {}, isTest = fa
  * @return {Promise<Object>} new record in company_contacts
 */
 const addContact = async (companyId, contactId, roleName, data = {}, isTest = false) => {
+  const roleId = await getRoleId(roleName);
```

Comment:
Should this be inside the `try` block?

Reply:
The scope is OK as the roleId is needed in the `catch` block below